bindings/java/src/org/sleuthkit/datamodel/OsAccount.java

```diff
@@ -512,6 +513,15 @@ public OsAccountStatus getOsAccountStatus() {
 		return Collections.unmodifiableList(osAccountAttributes);
 	}
 	
+	/**
```

Comment:
Is this what's been agreed upon with @bcarrier, @rcordovano  & @jayaramcs ?

I've added and deleted this method multiple times now, as we go back & forth between whether we want OsAccount/OsAccountRealm etc. to be POJOs or be more OO-ey classes with methods. 

If we have decided to add the update() method here then perhaps we should also add one to OsAccountRealm to be consistent.

Reply:
This is what @bcarrier specified in the story.

Reply:
That may have been from a previous design when we had this method on there or I was just being lazy and imprecise when I meant OsAccountManager.update().  Lets remove this method from here.

Reply:
@bcarrier It seems a little inconsistent that the user should call OsAccount.addAttributes to update the account list of attributes, but OsManager to updated the reset of the object.

There is an odd inconsistent behavior here.